packages/components/src/Badge/_docs/Badge.stickersheet.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import { Badge } from "../index"
+
+export default {
+  title: "Components/Badge",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+    a11y: {
+      config: {
+        rules: [
+          {
+            // I think reverse stickersheets are being confused.
+            id: "color-contrast",
+            enabled: false,
+          },
+        ],
+      },
+    },
```

Comment:
If it's only a problem with the reversed stickersheet, you can add this to the reversed story only :)

Reply:
Comment is wrong, it's all stickersheets